server/events/vcs/gitlab_client.go

```diff
@@ -441,10 +441,13 @@ func (g *GitlabClient) UpdateStatus(logger logging.SimpleLogging, repo models.Re
 	var (
 		resp        *gitlab.Response
 		maxAttempts = 10
-		b           = &backoff.Backoff{Jitter: true}
+		retryer     = &backoff.Backoff{
+			Jitter: true,
+			Max:    g.PollingInterval,
```

Comment:
the default `Max` is 30s per attempt, which I felt was excessive; so using the polling interval clamps us at 1s per attempts maximum; which according to our internal tests should be more than fine